### PR TITLE
webui: when multiple devices have the same name disable mount point mapping scenario

### PR DESCRIPTION
**Pull request by KKoukiou**
This is an unsupported for the time being.

### CODE REVIEW

File: ui/webui/src/components/storage/InstallationMethod.jsx [hunk 1]
```diff
@@ -612,10 +612,11 @@ export const InstallationMethod = ({
                 />
                 <InstallationScenario
                   deviceData={deviceData}
-                  diskSelection={diskSelection}
+                  ndiskSelection={diskSelection}
```
Comment (rvykydal) [inline ui/webui/src/components/storage/InstallationMethod.jsx:615]: Some lost keystroke I guess... ?

**Comment by rvykydal** [comment]
After fixing the typo works for me. 
Maybe we could add also fixing the issue (like `btrfs filesystem label mounpoint uniquelabel`) to the test?

**Review by rvykydal — APPROVED** [review]

**Review by vojtechtrefny — COMMENTED** [review]
I think we should show the error only on mount point assignment or maybe only disable mount point assignment. Erase & install should work in this situation and using only free space also shouldn't have issue with duplicate devices.

File: ui/webui/src/components/storage/InstallationMethod.jsx (hunk 1 above)
Comment (KKoukiou) [reply ui/webui/src/components/storage/InstallationMethod.jsx]: tnx fixed!

**Review by rvykydal — APPROVED** [review]
Looks great to me.

**Comment by KKoukiou** [comment]
Lets' ask @garrett for the helper message review.

**Review by vojtechtrefny — APPROVED** [review]
Looks good to me.

**Comment by garrett** [comment]
Issues:

- "To enable" sounds weird to me. But, it would need a comma after it "To enable, do foo". "To use this option" sounds better than "To enable", as the goal isn't enabling, but to use the installation method corresponding with the option.
- When using sentences, the UI should always use sentences. There's a period missing. Either fully use sentences in the UI, or use phrases; do not mix. UI should stay consistent. (In other words: There's a missing period.)
- What is "free up disk space"? How do you do that from this UI? How much do you need? It's not clear.
- We wouldn't need "the following".
- "This option requires that the selected devices has formatted partitions." is so wrong in many ways. It shouldn't lead with "This option" in the description; "requires" doesn't make sense here either. It should explain what the purpose of the option is instead.

Suggestions:

- Only is in the wrong place; this should be better: "Keep current disk layout and only install into available space."
- "Not enough free space. To use this option, resize or remove existing partitions to free up at least 60 gigabytes." (Adjust for whatever the actual requirement is.)
- Mount point assignment should have this explanation text: "Select existing formatted partitions for installation."
- "Some devices use the same name: XYZ. To use this option, rename partitions to have unique names."

**Review by rvykydal — APPROVED** [review]
Looks good to me, I just noticed that the "Some devices use the same name: XYZ" part of @garrett's suggestion was not applied but maybe it is on purpose.

**Review by vojtechtrefny — APPROVED** [review]

**Review by rvykydal — APPROVED** [review]

**Pull request merged**